web/web.js

```diff
@@ -1,10 +1,34 @@
+import { user_info_wos } from "db";
 import express from "express";
 import { hybridServer } from "hybrid-http-server";
 
+
 const app = express();
 
 // Static file serving
 const __dirname = new URL(".", import.meta.url).pathname;
 app.use(express.static(__dirname + "/static"));
 
+/**
+ * /wos/users:
+ *      get:
+ *          list of 10 users with most debt and more than -5$ with the following format:
+ *           [
+ *             {
+ *               "id": 2,
+ *               "username": "visitday",
+ *               "balance": 0
+ *             },
+ *             {
+ *               "id": 1,
+ *               "username": "bob",
+ *               "balance": 1337
+ *             }
+ *           ]
+*/
+app.get('/wos/users', async (req, res) => {
```

Comment:
For the sake of our sanity, we should put all api endpoints under `/api` so we can avoid any conflicts with static files. There's a nice way with `express` to do this using routers like so:

```js

const api = express.Router();
api.get('/wos/users', /*...*/);

app.use('/api', api);
```

That would make the final HTTP endpoint `/api/wos/users`. That should be all you need but if you want / need more here's the info routers: https://expressjs.com/en/5x/api.html#router